packages/repeater/src/lib/Repeater.ts

```diff
@@ -81,4 +116,20 @@ export class Repeater {
       })
     );
   }
+
+  private setupShutdown(): void {
+    ['SIGTERM', 'SIGINT', 'SIGHUP'].forEach(event =>
+      process.on(event, async () => {
+        if (this.runningStatus === RunningStatus.RUNNING) {
+          try {
+            await this.stop();
+            process.exit(0);
+          } catch (e) {
+            this.logger?.error(e.message);
+            process.exit(1);
+          }
```

Comment:
I don't think that repeater should close a process. The client code (e.g. jest) is in charge of that 
```suggestion
          try {
            await this.stop();
          } catch (e) {
            this.logger?.error(e.message);
          }
```